src/docfx/build/xref/XrefMap.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace Microsoft.Docs.Build
+{
+    public class XrefMap
```

Comment:
>XrefMap [](start = 17, length = 7)

keep consistency: `Xref` or `XRef`?

Reply:
should be XRef

---
In reply to: [218288201](https://github.com/dotnet/docfx/pull/3428#discussion_r218288201) [](ancestors = 218288201)